recipes/rios/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+python setup.py install --prefix=$PREFIX
```

Comment:
If not, I think we can remove both the build.sh and the bld.bat and replace with a ```build/script``` section.